asserts/database.go

```diff
@@ -129,14 +129,18 @@ type RODatabase interface {
 // assertions in the database.
 type Checker func(assert Assertion, signingKey *AccountKey, roDB RODatabase, checkTime time.Time) error
 
+// A NoAuthorityChecker defines a check on a no-authority assertion.
+type NoAuthorityChecker func(assert Assertion, roDB RODatabase) error
```

Comment:
Just wondering if it's useful to have a `checkTime` here too, similar to a Checker? For example, it could be used to ensure that the range of an AccountKeyRequest's `since` and `until` headers are valid.